dvc/testing/fixtures.py

```diff
@@ -215,48 +213,34 @@ def local_workspace(make_workspace):
     return make_workspace("workspace", typ="local")
 
 
-@pytest.fixture(scope="session")
-def docker():
-    # See https://travis-ci.community/t/docker-linux-containers-on-windows/301
-    if os.environ.get("CI") and os.name == "nt":
-        pytest.skip("disabled for Windows on Github Actions")
-
-    try:
-        subprocess.check_output("docker ps", shell=True)
-    except (subprocess.CalledProcessError, OSError):
-        pytest.skip("no docker installed")
-
-
-@pytest.fixture(scope="session")
-def docker_compose(docker):  # noqa: ARG001
-    try:
-        subprocess.check_output("docker-compose version", shell=True)
-    except (subprocess.CalledProcessError, OSError):
-        pytest.skip("no docker-compose installed")
-
-
 @pytest.fixture(scope="session")
 def docker_compose_project_name():
     return "pytest-dvc-test"
 
 
 @pytest.fixture(scope="session")
-def docker_services(docker_compose_file, docker_compose_project_name, tmp_path_factory):
-    # overriding `docker_services` fixture from `pytest_docker` plugin to
-    # only launch docker images once.
+def docker_services(request, tmp_path_factory):
+    if os.environ.get("CI") and os.name == "nt":
+        pytest.skip("disabled for Windows on CI")
 
-    from filelock import FileLock
+    try:
+        subprocess.check_output("docker ps", stderr=subprocess.STDOUT, shell=True)
+    except subprocess.CalledProcessError as err:
+        out = (err.output or b"").decode("utf-8")
+        pytest.skip(f"docker is not installed or the daemon is not running: {out}")
 
-    # pylint: disable-next=import-error
-    from pytest_docker.plugin import DockerComposeExecutor, Services
+    try:
+        cmd = "docker-compose version"
+        subprocess.check_output(cmd, stderr=subprocess.STDOUT, shell=True)
+    except subprocess.CalledProcessError as err:
+        out = (err.output or b"").decode("utf-8")
+        pytest.skip(f"docker-compose is not installed: {out}")
 
-    executor = DockerComposeExecutor(docker_compose_file, docker_compose_project_name)
```

Comment:
@skshetry Why was this removed?

Reply:
Looks like this fixture is not doing anything now.

Reply:
https://github.com/iterative/dvc/pull/9083

Reply:
There was no reason to depend on the internals of `pytest_docker`.

Reply:
@skshetry I don't understand. With the code removed this fixture does nothing.

Reply:
We return the parent fixture here (after checking whether we should skip or not):

https://github.com/iterative/dvc/blob/b80d84e689783ef92fa666acfbeb13dfd26cd861/dvc/testing/fixtures.py#L256